security-proxy/targets-mapping.properties

```diff
@@ -1,8 +1,10 @@
-analytics=http://localhost:8280/analytics/
-atlas=http://localhost:8280/atlas/
-console=http://localhost:8280/console/
-extractorapp=http://localhost:8280/extractorapp/
-geonetwork=http://localhost:8280/geonetwork/
-geoserver=http://localhost:8380/geoserver/
-header=http://localhost:8280/header/
-mapfishapp=http://localhost:8280/mapfishapp/
+analytics=http://analytics:8080/analytics/
+atlas=http://atlas:8080/atlas/
+console=http://console:8080/console/
+extractorapp=http://extractorapp:8080/extractorapp/
+geonetwork=http://geonetwork:8080/geonetwork/
+geoserver=http://geoserver:8080/geoserver/
+header=http://header:8080/header/
+mapfishapp=http://mapfishapp:8080/mapfishapp/
+geowebcache=http://geowebcache:8080/geowebcache/
+mapstore=http://mapstore:8080/mapstore/
```

Comment:
Same here, you should keep using `localhost`
Sorry, missing my morning coffee ! ...